src/Scene.test.js

```diff
@@ -0,0 +1,92 @@
+// @flow
+
+import React from 'react';
+import Adapter from 'enzyme-adapter-react-16';
+import { configure, mount } from 'enzyme';
+import { IntlProvider } from 'react-intl';
+import messages from './messages.json';
+import Scene from './Scene';
+
+configure({ adapter: new Adapter()});
+
+const mockCharacterState : any = {
+    xPos: 0,
+    yPos: 0,
+    directionDegrees: 90
+};
+
+const defaultSceneProps = {
+    numRows: 1,
+    numColumns: 1,
+    gridCellWidth: 10
+};
+
+function createMountScene(props) {
+    const wrapper = mount(
+        React.createElement(
+            Scene,
+            Object.assign(
+                {},
+                defaultSceneProps,
+                {
+                    characterState: mockCharacterState
+                },
+                props
+            )
+        ),
+        {
+            wrappingComponent: IntlProvider,
+            wrappingComponentProps: {
+                locale: 'en',
+                defaultLocale: 'en',
+                messages: messages.en
+            }
+        }
+    );
+
+    return wrapper;
+}
+
+function findGridLines(sceneWrapper) {
+    return sceneWrapper.find('.Scene__grid-line');
+}
+
+function findGridLabels(sceneWrapper) {
+    return sceneWrapper.find('.Scene__grid-label');
+}
+
+function findCharacter(sceneWrapper) {
+    return sceneWrapper.find('.RobotCharacter');
+}
+
+describe('When the Scene renders', () => {
```

Comment:
As we discussed on Skype, please extend the tests to check the gird line and label coordinates and the position (transform) of the character.